include/aspect/postprocess/gravity_point_values.h

```diff
@@ -149,67 +149,66 @@ namespace aspect
         double quadrature_degree_increase;
 
         /**
-         * Parameter for the map sampling scheme:
+         * Parameter for the uniform distribution sampling scheme:
          * Gravity may be calculated for a sets of points along the radius (e.g. depth
          * profile) between a minimum and maximum radius. Number of points along the radius
          * is specified with n_points_radius.
          */
         unsigned int n_points_radius;
 
         /**
-         * Parameter for the map sampling scheme:
+         * Parameter for the uniform distribution sampling scheme:
          * Gravity may be calculated for a sets of points along the longitude (e.g. satellite
          * mapping) between a minimum and maximum longitude. Number of points along the
          * longitude is specified with n_points_longitude.
          */
         unsigned int n_points_longitude;
 
         /**
-         * Parameter for the map sampling scheme:
+         * Parameter for the uniform distribution sampling scheme:
          * Gravity may be calculated for a sets of points along the latitude (e.g. satellite
          * mapping) between a minimum and maximum latitude. Number of points along the
          * latitude is specified with n_points_latitude.
          */
         unsigned int n_points_latitude;
 
         /**
-         * Parameter for the map sampling scheme:
-         * Minimum radius for the depth range in case of the map sampling scheme. Presribe
-         * a minimum radius for a sampling coverage at a specific height. May be defined
-         * in or outside the model.
+         * Parameter for the uniform distribution sampling scheme:
+         * Presribe a minimum radius for a sampling coverage at a specific height.
+         * May be defined in or outside the model.
          */
         double minimum_radius;
 
         /**
-         * Parameter for the map sampling scheme:
-         * Maximum radius for depth-profile in case of the map sampling scheme. May be
+         * Parameter for the uniform distribution sampling scheme:
+         * Maximum radius may be used for instance for depth-profile. May be
```

Comment:
This is not clear to me. Do you mean 'may be used for the uniform sampling scheme'? You already mention that in the last sentence. Or do you mean 'if n_points_radius is more than one'?